app/assemblers/assemble_representation_tracks_for_select.rb

```diff
@@ -20,9 +20,16 @@ def supermentored_tracks
     Track.where(id: mentor.track_mentorships.supermentor_frequency.select(:track_id)).order(title: :asc)
   end
 
+  def current_mentor
+    return nil if is_admin
+
+    mentor
+  end
```

Comment:
I'm not sure about the naming. How is `current_mentor` different from mentor? Maybe it might make sense to inline this inside the `track_num_representations` method:

`num_representations_mentor = is_admin ? nil : mentor`

Or something like that